src/rpc/rawtransaction.cpp

```diff
@@ -914,6 +914,10 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
     if (request.params.size() > 2)
         fInstantSend = request.params[2].get_bool();
 
+    bool fBypassLimits = false;
+    if (request.params.size() > 3)
+        fBypassLimits = request.params[3].get_bool();
```

Comment:
Doesn't this need an addition in https://github.com/dashpay/dash/blob/9451782a0496e99d5f2f4b91716a36b693ebc09d/src/rpc/client.cpp#L112?